tf/tfprocess.py

```diff
@@ -706,10 +847,11 @@ def construct_net(self, planes):
         self.weights.append(W_fc2)
         self.weights.append(b_fc2)
         h_fc2 = tf.nn.relu(tf.add(tf.matmul(h_conv_val_flat, W_fc2), b_fc2))
-        W_fc3 = weight_variable([128, 1], name='fc3/weight')
-        b_fc3 = bias_variable([1], name='fc3/bias')
+        value_outputs = 3 if self.wdl else 1
+        W_fc3 = weight_variable([128, value_outputs], name='fc3/weight')
+        b_fc3 = bias_variable([value_outputs], name='fc3/bias')
         self.weights.append(W_fc3)
         self.weights.append(b_fc3)
-        h_fc3 = tf.nn.tanh(tf.add(tf.matmul(h_fc2, W_fc3), b_fc3), name='value_head')
```

Comment:
Shouldn't this be conditional on self.wdl - tanh if not wdl otherwise tanh?

Reply:
Yes, it should have tanh when not using WDL. Apparently it works close enough even without it and I didn't notice it in testing.